### DISCUSSION TITLE
Forwarding to Jetson Nano

### DISCUSSION THREAD

**Discussion by elifozisik** [svpcom/wfb-ng#221] [Questions]
Hi, 

I use 2 Raspberry Pi 3 and one jetson nano. I connect ground station Raspberry Pi to jetson nano via ethernet. And I want to receive the drone video from jetson nano? How can i forward the video to Jetson Nano? Have you any idea?

Thanks.

**Comment by svpcom** [comment]
Add to `/etc/wifibroadcast.cfg` on GS:
```
[gs_video]
peer = 'connect://X.X.X.X:5600'
```

where `X.X.X.X` is ip-address of the Jetson Nano.
Then run gstreamer pipeline which will listen on udp 5600 and decode rtp-h264 stream on jetson nano.

**Reply by elifozisik** [reply]
I try to this command in terminal but i got an error `Spinning.  Please run 'gdb gst-launch-1.0 21982' to continue debugging, Ctrl-C to quit, or Ctrl-\ to dump core.
`
Command
```
gst-launch-1.0 udpsrc port=5600 caps='application/x-rtp, media=(string)video, clock-rate=(int)90000, encoding-name=(string)H264' \
               ! rtph264depay ! avdec_h264 ! clockoverlay valignment=bottom ! autovideosink fps-update-interval=1000 sync=false
```
screenshot from `/etc/wifibroadcast.cfg`

![image](https://user-images.githubusercontent.com/51250917/169715202-96b22459-ebf3-4d76-b7e0-f13adc26f6d7.png)

**Reply by svpcom** [reply]
Have you restart service after config file modify?

**Reply by elifozisik** [reply]
yes

**Comment by svpcom** [comment]
Try this:
```
gst-launch-1.0 udpsrc do-timestamp=true port=5600 caps='application/x-rtp' ! rtph264depay ! h264parse disable-passthrough=true ! avdec_h264 ! xvimagesink sync=false
```

but it is not accelerated and may be slow on jetson. You need to found pipeline which use OMX if this example succeeds

**Reply by elifozisik** [reply]
I tried that command but didn't happen

![image](https://user-images.githubusercontent.com/51250917/169717721-620aa56f-801d-4344-ac68-e80f2ded839f.png)

**Reply by svpcom** [reply]
run `wfb-cli gs` on the GS

**Reply by elifozisik** [reply]
here is screenshot

![image](https://user-images.githubusercontent.com/51250917/169717938-002921a6-8f3a-4a32-9b62-5fb161a62b58.png)

**Reply by svpcom** [reply]
check with tcpdump that jetson receive video stream from GS. According to `wfb-cli` GS works correctly

**Reply by reedev** [reply]
Isn’t this since both services need to be started on gs, as described in another discussion? (Not sure which discussion)
